### PR TITLE
docs: Add to glossary: L1, L2, Starknet

### PR DESCRIPTION
**Pull request by stoobie**
Add to glossary: L1, L2, (L3, commented out for now) and Starknet

### CODE REVIEW

File: modules/ROOT/partials/partial_glossary.adoc [hunk 1]
```diff
@@ -71,6 +71,22 @@ The smallest denomination of STRK. 1 fri = 10^-18^ STRK.
...
 
+[#l1]
+== Layer 1, L1
+
+The base layer of a blockchain network, such as Ethereum, where transactions are directly recorded and validated. It includes the underlying protocol and consensus mechanism that establish the foundational rules for the blockchain.
+
+[#l2]
+== Layer 2, L2
+
+Exists on top of an L1 blockchain. Starknet is an L2 blockchain for Ethereum, enabling any dApp to achieve massive achieve massive computation scale without compromising on Ethereum’s composability and security.
```
Comment (0xLucqs) [inline modules/ROOT/partials/partial_glossary.adoc]:
```suggestion
Exists on top of an L1 blockchain. Starknet is an L2 blockchain for Ethereum, enabling any dApp to achieve massive computation scale without compromising on Ethereum’s composability and security.
```

**Review by 0xLucqs — CHANGES_REQUESTED** [review]

File: modules/ROOT/partials/partial_glossary.adoc [hunk 2]
```diff
@@ -71,6 +71,22 @@ The smallest denomination of STRK. 1 fri = 10^-18^ STRK.
...
 
+[#l1]
+== Layer 1, L1
+
+The base layer of a blockchain network, such as Ethereum, where transactions are directly recorded and validated. It includes the underlying protocol and consensus mechanism that establish the foundational rules for the blockchain.
+
+[#l2]
+== Layer 2, L2
+
+Exists on top of an L1 blockchain. Starknet is an L2 blockchain for Ethereum, enabling any dApp to achieve massive computation scale without compromising on Ethereum’s composability and security.
```
Comment (stoobie) [inline modules/ROOT/partials/partial_glossary.adoc]:
```suggestion
Exists on top of an L1 blockchain, such as Ethereum, in order to add value to the L1, by providing features such as massive computation scale and security. Starknet is an L2 blockchain that exists on Ethereum.
```

File: modules/ROOT/partials/partial_glossary.adoc [hunk 3]
```diff
@@ -71,6 +71,22 @@ The smallest denomination of STRK. 1 fri = 10^-18^ STRK.
...
 
+[#l1]
+== Layer 1, L1
+
+The base layer of a blockchain network, such as Ethereum, where transactions are directly recorded and validated. It includes the underlying protocol and consensus mechanism that establish the foundational rules for the blockchain.
+
+[#l2]
+== Layer 2, L2
+
+Exists on top of an L1 blockchain, such as Ethereum, in order to add value to the L1, by providing features such as massive computation scale and security. Starknet is an L2 blockchain that exists on Ethereum.
```
Comment (0xLucqs) [inline modules/ROOT/partials/partial_glossary.adoc]: It doesn't bring security it scales ethereum without losing any security

**Review by 0xLucqs — CHANGES_REQUESTED** [review]

File: modules/ROOT/partials/partial_glossary.adoc (hunk 3 above)
Comment (stoobie) [reply modules/ROOT/partials/partial_glossary.adoc]: @LucasLvy 
I thought that scaling computation is the purpose of the rollup, but that the point of STARK proofs is to prove the validity of transactions, which sounds like a form of security, over and above what Ethereum provides.

Comment (stoobie) [inline modules/ROOT/partials/partial_glossary.adoc]:
```suggestion
Exists on top of an L1 blockchain, such as Ethereum, in order to add value to the L1, by providing features such as massive computation scale without compromising on Ethereum’s composability and security.
```

**Review by 0xLucqs — APPROVED** [review]

**Pull request merged**